modules/site_config.py

```diff
@@ -5,6 +5,9 @@
 from modules import util
 from string import Template
 
+# ATT&CK version
+attack_version = "8.0"
```

Comment:
This can probably be pulled from [data/versions.json](https://github.com/mitre-attack/attack-website/blob/master/data/versions.json#L3). Fine with keeping it here if there's a good reason for it, but otherwise it's better to adhere to good DRY principles.